internal/pkg/manifest/backend_svc.go

```diff
@@ -100,7 +100,15 @@ func (s *BackendService) Port() (port uint16, ok bool) {
 
 // Publish returns the list of topics where notifications can be published.
 func (s *BackendService) Publish() []Topic {
-	return s.BackendServiceConfig.PublishConfig.Topics
+	var pubs []Topic
+	for _, topic := range s.BackendServiceConfig.PublishConfig.Topics {
```

Comment:
nit: instead of replicating the same body across all service types, I think in this scenario the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle is valid.

We could move this method to 

```
// publishedTopics augments each  topic to append the .fifo suffix if the topic is FIFO, and returns their configuration.
func (cfg PublishConfig) publishedTopics() []Topic {
  ...
}
```